theseus/labs/lie/functional/se3_impl.py

```diff
@@ -81,19 +81,9 @@ def rand(
 ) -> torch.Tensor:
     if len(size) != 1:
         raise ValueError("The size should be 1D.")
-    rotation = so3.rand(
-        size[0],
-        generator=generator,
-        dtype=dtype,
-        device=device,
-    )
+    rotation = so3.rand(size[0], generator=generator, dtype=dtype, device=device)
```

Comment:
For consistency with the new user facing `lieF.SE3/SO3` should the imports internally also update to SE3/SO3 (line 10 in this file)?

Reply:
Sounds good, will fix.